lib/DDGC/Config/Subscriptions.pm

```diff
@@ -19,18 +19,27 @@ sub created_ia_page {
     };
 }
 
+sub created_ia_page {
+    my ( $self, $meta ) = @_;
+    $self->_subscription(
+        'instant_answer',
+        'created',
+        $meta,
+    );
+}
+
 sub updated_ia_page {
     my ( $self, $meta ) = @_;
-    +{
-        category     => 'instant_answer',
-        action       => 'updated',
-        meta1        => $meta->{meta1} // undef,
-        meta2        => $meta->{meta2} // undef,
-        meta3        => $meta->{meta3} // undef,
-        ( $meta->{description} )
-            ? ( description  => $meta->{description} )
-            : (),
-    };
+    $self->_subscription(
+        'instant_answer',
+        'updated',
+        $meta,
+    );
+}
+
+sub ia_page {
+    my ( $self, $id ) = @_;
+    $self->updated_ia_page( { meta1 => $id } );
```

Comment:
I keep going back and forth on whether or not we should have `id` as an intrinsic field instead. I suppose that speaks to the fact we don't need to change it...